### ISSUE TITLE
File selection highlight contrast

### ISSUE THREAD

**Issue by felipezacani** [Monokai/monokai-pro-vscode#112]
When I'm working on a file and I select any other, the tree shows this extreme contrast highlight of the file, not sure exactly why. Screenshot attached.
![image](https://user-images.githubusercontent.com/6606367/52492453-5e6dbf80-2bb0-11e9-8689-4dd4d5d2da58.png)

**Comment by en3sis** [comment]
Same issue after last VSC update.

**Comment by devert** [comment]
Until this gets fixed in the theme, you can override this color yourself in your `settings.json` with...

```
"workbench.colorCustomizations": {
    "list.inactiveFocusBackground": "#0000"
}
```

**Comment by en3sis** [comment]
@devert thank you! =)

**Comment by caiceA** [comment]
@devert  can i slightly change the background slightly darker? I am using this one octagon filter its ok but i want to change the background darker below is my current theme

any suggestion / help ?

<img width="1676" alt="screenshot 2019-02-18 at 12 16 58" src="https://user-images.githubusercontent.com/16766231/52947473-1b150d00-3377-11e9-9d93-bc96656e5c36.png">

**Comment by caiceA** [comment]
never mind i have made it. thanks

**Comment by Monokai** [comment]
Fixed in 1.1.10
